requirements.txt

```diff
@@ -9,4 +9,5 @@ openai==1.30.1
 pandas==2.2.2
 python-dotenv==1.0.1
 scikit_learn==1.4.2
-tiktoken
\ No newline at end of file
+tiktoken
+spaCy==3.7
```

Comment:
Why is this package is set to 3.7 since the latest is 3.7.4?

Reply:
Also you can add the spacy model to `requirements,txt`
You could simply add the model giving the URL, Does this work for you? I was able to install the spaCy pkg and model
`https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.7.1/en_core_web_sm-3.7.1.tar.gz`

Reply:
I am trying to simply the process for getting all the package requirements via the standard file.

Reply:
I think there is a conflict with `spaCy = 3.7.0` version with `en_core_web_sm-3.7.1`

Reply:
Updated the requirements in the new commit. 
I noticed that I needed to have C++ build tools installed when building some of the wheels for package installation. Is this something I need to specify in the ReadMe? I think it is a Windows requirement, and not sure how it might appear on other OSes.